### PR TITLE
[carousel] Prevent throwing exception on non exsisting files

### PR DESCRIPTION
**Pull request by zorn-v**
Useful when develop on different machines with the same database

### CODE REVIEW

File: local/modules/Carousel/Loop/CarouselLoop.php [hunk 1]
```diff
@@ -73,6 +73,10 @@ public function parseResults(LoopResult $loopResult)
     {
         /** @var \Carousel\Model\Carousel $carousel */
         foreach ($loopResult->getResultDataCollection() as $carousel) {
+            if (!file_exists($carousel->getUploadDir() . DS . $carousel->getFile())) {
+                continue;
```
Comment (roadster31) [inline local/modules/Carousel/Loop/CarouselLoop.php]: Adding a warning or an error in the Thelia log would be a nice idea.

**Comment by roadster31** [comment]
Could you please rebase ?

**Comment by roadster31** [comment]
Sorry, I'm afraid you have to rebase again. Please ping me when you're done.

**Comment by zorn-v** [comment]
Merged

**Comment by roadster31** [comment]
Thanks.

**Pull request merged**